### PR TITLE
Neotheology “Spark” and “Radiance” Power Cells

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Added cheap NT-branded non-rechargeable power cells, they come in two flavors and all sizes.

NeoTheology "Spark 300S" (1 steel, 10 biomatter)
NeoTheology "Spark 1000M" (2 steel, 20 biomatter)
NeoTheology "Spark 13000L" (3 steel, 30 biomatter)

NeoTheology "Radiance 500S" (1 steel, 5 biomatter, 1 plasma)
NeoTheology "Radiance 1500M" (2 steel, 10 biomatter, 2 plasma)
NeoTheology "Radiance 20000L" (3 steel, 15 biomatter, 3 plasma)

Those spawn fully charged, but cannot be recharged once spent, only recycling and printing new one.
When recycled, spark-series lose half biomatter, and radiance lose all plasma.
~~Can be printed from 'NeoTheology Bioprinter Products and Utilities'.~~
Can be printed from 'NeoTheology Armory - Power Cells Pack', obtainable via EOTP.

image

Comes from [this document](https://docs.google.com/document/d/1DECLfZjtbj92sTkSON6T3fstegJqYVbjtLu3-6WvTiA/edit).

## Why It's Good For The Game

NT no longer have to rely on moebius or junking to get power cells.
Finally some use for plasma.

## Changelog
:cl:
add: two types of NT-branded cells, that cannot be recharged
imageadd: sprites for neoteo cells

/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
Is this effectively free and infinity power given you can recycle the cells and print them again?

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 1]
```diff
@@ -25,6 +25,13 @@
 		/datum/design/autolathe/device/grenade/nt_cleaner,
 		/datum/design/autolathe/device/grenade/nt_weedkiller,
 
+		/datum/design/bioprinter/nt_cells/large,
+		/datum/design/bioprinter/nt_cells/large/plasma,
+		/datum/design/bioprinter/nt_cells/medium,
+		/datum/design/bioprinter/nt_cells/medium/plasma,
+		/datum/design/bioprinter/nt_cells/small,
+		/datum/design/bioprinter/nt_cells/small/plasma,
```
Comment (Firefox13) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:33]: Might be a better idea to put them as EOTP reward instead.

**Comment by SirRichardFrancis** [comment]
> Is this effectively free and infinity power given you can recycle the cells and print them again?

Not free exactly, some resources lost when recycling. Not a big deal with biomatter, but plasma is somewhat rare, and putting it in packman would be much more effective.

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm (hunk 1 above)
Comment (SirRichardFrancis) [reply code/game/objects/items/weapons/design_disks/NeoTheology.dm:33]: That will make them unavailable most of the time tho.

**Comment by SirRichardFrancis** [comment]
image
image

**Comment by FortuneMaster** [comment]
increase the cost of biomatter

**Comment by SirRichardFrancis** [comment]
> increase the cost of biomatter

By how much? And how much of that biomatter should be non-refundable? Exact numbers for each cell would be good.

**Comment by SirRichardFrancis** [comment]
Changed values to my liking, description updated. Sparks lose half biomatter, radiance lose all plasma.

**Comment by FortuneMaster** [comment]
yeah, that pricing is better.

**Review by TheShown911 — APPROVED** [review]
Looks good, would check their export prices just in case as I remember power cells using a different formula.

**Pull request merged**